src/components/common/ClassroomEditor.jsx

```diff
@@ -0,0 +1,240 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Actions } from 'jumpstate';
+import { saveAs } from 'browser-filesaver';
+import { blobbifyData, generateFilename } from '../../lib/mapexplorer-helpers'; //TODO: Maybe not brand this as 'mapexplorer'?
+
+import Anchor from 'grommet/components/Anchor';
+import Box from 'grommet/components/Box';
+import Button from 'grommet/components/Button';
+import Heading from 'grommet/components/Heading';
+import Label from 'grommet/components/Label';
+import Layer from 'grommet/components/Layer';
+import List from 'grommet/components/List';
+import ListItem from 'grommet/components/ListItem';
+import Paragraph from 'grommet/components/Paragraph';
+import Table from 'grommet/components/Table';
+import TableRow from 'grommet/components/TableRow';
+import Toast from 'grommet/components/Toast';
+
+import CloseIcon from 'grommet/components/icons/base/Close';
+import EditIcon from 'grommet/components/icons/base/Edit';
+import Spinning from 'grommet/components/icons/Spinning';
+import LinkPreviousIcon from 'grommet/components/icons/base/LinkPrevious';
+import ClassroomCreateFormContainer from '../../containers/common/ClassroomCreateFormContainer';
+import CopyToClipboard from 'react-copy-to-clipboard';
+
+import {
+  CLASSROOMS_STATUS, CLASSROOMS_INITIAL_STATE, CLASSROOMS_PROPTYPES,
+} from '../../ducks/classrooms';
+import {
+  ASSIGNMENTS_STATUS,
+} from '../../ducks/assignments';
+
+class ClassroomEditor extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+  
+  render() {
+    const props = this.props;
+    if (!props.selectedClassroom) return null;
+    this.exportGrades = this.exportGrades.bind(this);
+
+    const joinURL = `https://${window.location.host}/students/classrooms/join?id=${props.selectedClassroom.id}&token=${props.selectedClassroom.joinToken}`;
```

Comment:
This may need to go into the central config file eventually, but I'm curious to know how we'd handle different projects. (e.g. a join screen for Intro 2 Astro and another for WildCam Darien? If not, how would students be redirected to the appropriate project page after they've signed up?)